README.md

```
@@ -356,15 +434,19 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
 
 Once your first push notifications are sent and opened, you should start to see _Opened Push_ metrics within your Klaviyo dashboard.
 
-#### Rich Push (Images & Videos)
+For runnable reference examples, see [SPMExample](Examples/KlaviyoSwiftExamples/SPMExample) or [CocoapodsExample](Examples/KlaviyoSwiftExamples/CocoapodsExample).
+
+---
+
+### Rich Push (Images & Videos)
 
 >  ℹ️ Rich push notifications are supported in SDK version [2.2.0](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/2.2.0) and higher
```

markdownlint-cli2 (0.23.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)